src/pages/Home.vue

```diff
@@ -104,12 +105,12 @@ export default {
   },
   computed: {
     getDocumentPageLink() {
```

Comment:
Check if this is really required @jinalviranii

Reply:
Yes this is needed